bin/tx-prover/src/commands/proxy.rs

```diff
@@ -65,6 +65,13 @@ impl StartProxy {
         http_server_options.h2c = true;
         logic.server_options = Some(http_server_options);
 
+        // Enable Prometheus metrics
+        let mut prometheus_service_http =
+            pingora::services::listening::Service::prometheus_http_service();
+        prometheus_service_http
+            .add_tcp(format!("{}:{}", "127.0.0.1", proxy_config.prometheus_port).as_str());
```

Comment:
Related to the previous comment: is there a reason to hard code `127.0.0.1` here rather than use something like `prometheus_endpoint` config?

Reply:
Yes, I changed it because I thought about it didn't realize of use cases of allowing host different than `localhost` for the metrics. But now giving a second thought, I doesn't make sense. We might want to use it another host, and we can use `localhost` as the default.

Reply:
I will add the exporter host  as variable back.